05/implement-network-delay.instructor.md

```diff
@@ -0,0 +1,47 @@
+# Instructor: Implement Network Delay
+
+An example of a working implementation:
+
+```python
+import collections
+from heapq import heappush, heappop
+def network_delay_time(times, n, source):
+    # Initialize dictionary with default value as a list
```

Comment:
Folks got so hung up on default dict last cohort. Should we link them to the documentation?